code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -167,40 +167,43 @@ emp_act
 
 /mob/living/carbon/human/emp_act(severity)
 	if(isipc(src))
-		var/obj/item/organ/surge/s = src.internal_organs_by_name["surge"]
-		if(!isnull(s))
-			if(s.surge_left >= 1)
-				playsound(src.loc, 'sound/magic/LightningShock.ogg', 25, 1)
-				s.surge_left -= 1
-				if(s.surge_left)
-					visible_message("<span class='warning'>[src] was not affected by EMP pulse.</span>", "<span class='warning'>Warning: EMP detected, integrated surge prevention module activated. There are [s.surge_left] preventions left.</span>")
+		if(src.species == "Hunter-Killer")

```

Comment:
This is really badly done, I recommend either changing their organs/limbs emp coef or giving them the surger organ to have some protection from emps.

Reply:
They were intended to use surge module for a finite EMP protection